src/components/Main.js

```diff
@@ -1,22 +1,32 @@
 import {Component} from 'react';
 import HornedBeast from './HornedBeast.js';
-import data from './data.json';
 import {Container, Row, Col} from 'react-bootstrap';
+// import SelectedBeast from './SelectedBeast.js';
 
 class Main extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      // showModal: false
+    };
+  }
+
+
   render() {
-    let newBeasts = data.map(beast => {
+    let newBeasts = this.props.imageData.map(beast => {
```

Comment:
Great work on lab so far by the way!

Feedback here is to remove variable declaration of newBeasts. You have your imgData coming from props so you can just use this.props.imageData.map 👍